client/packages/common/src/ui/layout/tables/components/Expand/MiniTable.tsx

```diff
@@ -42,7 +42,7 @@ export const MiniTable = <T extends RecordWithId>({
             createQueryParamsStore<T>({ initialSortBy: { key: 'name' } })
           }
         >
-          <DataTable dense columns={columns} data={rows} key="mini-table" />
+          <DataTable dense columns={columns} data={rows} id="mini-table" />
```

Comment:
and that's most of the PR, just updating the key from `key` to `id` because it is a reserved word. works either way.. but complains if you use `key`